### DISCUSSION TITLE
[closed][fairtracks] How to handle versioning of the FAIRtracks standard?

### DISCUSSION THREAD

**Discussion by sveinugu** [fairtracks/fairtracks_standard#119] [Team Posts]
@jmfernandez @dzerbino ?

**Comment by jmfernandez** [comment]
I think that, on the JSON Schema side, the version should be part of the '$id' in the schema, for major, incompatible versions (v1, v2 and so on). And the '@schema' attribute should also be updated accordingly to it. For minor fixes, which are backward compatible, we can use a numerical optional field called "@revision", which can be used to check up to which revision on that major release the schema is able to validate.

Also, I recommend to separate the two roles of $id : the URI of the schema and fixed value for @schema; the physical location of the JSON Schema. That is needed for the former, as the different revisions on a major release are sharing the same $id, but we might want to have copies of them separated (at least, the superseded ones).

**Comment by sveinugu** [comment]
Added issue: https://github.com/fairtracks/fairtracks_standard/issues/23. Closing this

**Comment by sveinugu** [comment]
Added link to this in: https://github.com/fairtracks/fairtracks_standard/issues/55
